app/client/ui2018/cssVars.ts

```diff
@@ -559,11 +559,11 @@ export const theme = {
   widgetPickerPrimaryBg: new CustomProp('theme-widget-picker-primary-bg', undefined, 'white'),
   widgetPickerSecondaryBg: new CustomProp('theme-widget-picker-secondary-bg', undefined,
     colors.lightGrey),
-  widgetPickerItemFg: new CustomProp('theme-widget-picker-item-fg', undefined, colors.lightGrey),
+  widgetPickerItemFg: new CustomProp('theme-widget-picker-item-fg', undefined, 'unset'),
   widgetPickerItemSelectedBg: new CustomProp('theme-widget-picker-item-selected-bg', undefined,
-    colors.lightGrey),
+    colors.mediumGrey),
```

Comment:
Is this a change in color, or what is the motivation?

Reply:
I looked back at the code before dark mode and noticed we were using mediumGrey before; I noticed it on the darker panels of the widget picker where lightGrey was blending into the panel background, making it so you couldn't actually tell if an item was selected.